components/title/basic/src/index.scss

```diff
@@ -0,0 +1,48 @@
+@import '~@schibstedspain/theme-basic/lib/index';
+
+$c-title-basic: $c-white;
+$c-subtitle-basic: $c-white;
+
+$ff-roboto-sans-serif: 'Roboto', sans-serif; // Will be moved to nc-theme-basic
```

Comment:
Actually, the title element is an h1 and the subtitle is an h2. So it would be a bit redundant if we use the mixin as well.
I will keep the styles for h1 and h2 already defined in theme-basic, and the scss file will contain only the custom styles required:

```
.sui-TitleBasic {
  &-title {
    color: $c-title-basic-main-title;
    font-weight: $fw-semi-bold;
  }
  &-subtitle {
    color: $c-title-basic-subtitle;
    font-weight: $fw-regular;
  }
```